commands.json

```diff
@@ -0,0 +1,62 @@
+{
+    "Error": {
+        "opcode": 0,
+        "name": "Error",
```

Comment:
does anything use the "name" field?

Reply:
Actually no, but could be useful in the future or for debugging purposes :stuck_out_tongue_closed_eyes:

Reply:
It's redundant to have it both here and in the key